src/main/antora/modules/ROOT/pages/mongodb/mapping/mapping.adoc

```diff
@@ -54,23 +54,23 @@ The following outlines what field will be mapped to the `_id` document field:
 | `@Id` `String` x
 | `_id`
 
-| `@Field("x")` `@Id` `String` x
-| `_id`
+| `@Field("x")` `@Id` `String` y
+| `x`
```

Comment:
This is incorrect, `@Id` takes precedence. I will update the change during the merge.